File: tests/fixtures/vb/sample.vb
Namespace SampleApp

''' <summary>User service.</summary>
Public Class UserService
    Public Const MAX_RETRIES As Integer = 3

    Public Property Name As String

    Public Sub New()
    End Sub

    Public Function GetUser(userId As Integer) As String
        Return userId.ToString()
    End Function
End Class

Public Interface IAuthenticator
    Function Authenticate(token As String) As Boolean
End Interface

Public Delegate Function TokenValidator(token As String) As Boolean

End Namespace

File: tests/fixtures/csharp/sample.cs
using System;

namespace SampleApp;

/// <summary>
/// User service.
/// </summary>
public class UserService
{
    public const int MAX_RETRIES = 3;

    public string Name { get; set; } = "";

    public UserService()
    {
    }

    public string GetUser(int userId)
    {
        return userId.ToString();
    }
}

public interface IAuthenticator
{
    bool Authenticate(string token);
}

public record UserRecord(int Id, string Name);
